docs/extend/modular-resources/upload/_index.md

```diff
@@ -179,7 +175,17 @@ For more information, see the [`viam module` command](/manage/cli/#module)
 
 ## Update an existing module
 
-You can also use the [Viam CLI](/manage/cli/) to update an existing custom module in the Viam Registry.
+You can update an existing module in the [Viam Registry](https://app.viam.com/registry) in one of two ways:
+
+- [Upload a newer version of your module manually](#update-an-existing-module-using-the-viam-cli) using the [Viam CLI](/manage/cli/).
+- [Upload a newer version of your module automatically](#update-an-existing-module-using-a-github-action) as part of a continuous integration (CI) workflow, using a GitHub Action.
```

Comment:
phrasing slightly confusing bc it sounds like you are doing this once

might change to 'upload new versions' or 'automatically upload on release' or something

(but this is a nit)